tools/sidekick/app.js

```diff
@@ -415,6 +415,14 @@
           sk.showModal(`Publishing ${path}`, true);
           const resp = await sendPurge(config, path);
           if (resp.ok) {
+            if (path.endsWith('/')) {
+              // directory, also purge index(.html)
+              await sendPurge(config, `${path}index`);
```

Comment:
Can't we run those in parallel to speed up the process ?

Reply:
We could... the purge is pretty quick however, it's the re-fetch that can take time...

Reply:
Feels much faster indeed!